clang/test/dpct/thrust_unsupported.cu

```diff
@@ -0,0 +1,19 @@
+// UNSUPPORTED: cuda-8.0
+// UNSUPPORTED: v8.0
+// RUN: dpct -out-root %T/thrust_unsupported %s --cuda-include-path="%cuda-path/include" --usm-level=none
+// RUN: FileCheck --input-file %T/thrust_unsupported/thrust_unsupported.dp.cpp --match-full-lines %s
+#include <thrust/host_vector.h>
+#include <thrust/replace.h>
+
+namespace thrust {
+   void replace_copy_if(int a, int b, int c) {
+   }
+};
+
+int main() {
+// CHECK: /*
+// CHECK: DPCT1107:{{[0-9]+}}: Migration for this overload of thrust::replace_copy_if is not supported.
+// CHECK: */
+  thrust::replace_copy_if(1,2,3);

```

Comment:
I have not found define of replace_copy_if with 3 argument. could you double check?

Reply:
There is no thrust::replace_copy_if with 3 arguments.  This test ensures that dpct behaves well if it encounters a thrust overload that is not expected in APINamesThrust.inc.

Reply:
@LU-JOHN, Since thrust::replace_copy_if with 3 arguments does not exist in thrust lib, it will meet parse error during migration, suggest you choose a thrust algo API, which part of overload versions are still not supported by oneDPL team, e.g thrust::scatter_if.